src/sha2.c

```diff
@@ -921,8 +921,8 @@ void SHA512_Last(SHA512_CTX* context) {
 		*context->buffer = 0x80;
 	}
 	/* Store the length of input data (in bits): */
-	*(sha2_word64*)&context->buffer[SHA512_SHORT_BLOCK_LENGTH] = context->bitcount[1];
-	*(sha2_word64*)&context->buffer[SHA512_SHORT_BLOCK_LENGTH+8] = context->bitcount[0];
+	memcpy(&context->buffer[SHA512_SHORT_BLOCK_LENGTH], &context->bitcount[1], sizeof(&context->bitcount[1]));
```

Comment:
Do we need to use `&` on the operand of `sizeof` on this (and the next) line?

Reply:
Thanks! Sorry for the nitpick.

Reply:
No, my fault - and I'm glad you caught it :)